tests/manage/test_create_pvc_random_storage_class.py

```diff
@@ -50,17 +46,20 @@ def teardown():
     """
     global RBD_SECRET_OBJ, CEPHFS_SECRET_OBJ
     log.info("Deleting PVC")
-    assert pvc.delete_all_pvcs()
+    for pvc_name in PVC_OBJ:
+        assert pvc.delete_pvcs(pvc_name.name)
```

Comment:
delete_pvcs is getting a list and here you are passing string (pvc name).
Use list comprehension:
```python
pvc_names = [pvc.name for pvc in PVC_OBJ]  #  Also change PVC_OBJ as per above comment
assert pvc.delete_pvcs(pvc_names)
```